docs/README.md

```diff
@@ -85,11 +85,13 @@ for account in accounts:
 
 ### Additional Pager Methods
 
-In addition to the methods to facilitate pagination, the Pager class provides 3 helper methods:
+In addition to the methods to facilitate pagination, the Pager class provides 3 helper methods.
 
-1. [count](#count)
-2. [first](#first)
-3. [take](#take)
+1. [count](#id2)
```

Comment:
Might want to see why readthedocs markdown parser does this. https://recurly-client-python.readthedocs.io/en/latest/#id2